uavcan/driver/python_can.py

```diff
@@ -0,0 +1,165 @@
+#
+# Copyright (C) 2019 UAVCAN Development Team  <uavcan.org>
+#
+# This software is distributed under the terms of the MIT License.
+#
+# Author: Jos Vos <jos.vos@demcon.nl>
+#
+
+from __future__ import division, absolute_import, print_function, unicode_literals
+import os
+import errno
+import time
+import threading
+from logging import getLogger
+
+from .common import DriverError, TxQueueFullError, CANFrame, AbstractDriver
+from .timestamp_estimator import TimestampEstimator
+
+try:
+    import queue
+except ImportError:
+    # noinspection PyPep8Naming,PyUnresolvedReferences
+    import Queue as queue
+
+logger = getLogger(__name__)
+
+try:
+    import can
+except ImportError:
+    logger.info("no module 'can' available, please install python-can")
+    can = None
+
+
+class PythonCan(AbstractDriver):
+    TX_QUEUE_SIZE = 1000
+
+    def __init__(self, interface, **_extras):
+        super(PythonCan, self).__init__()
+
+        if can is None:
+            logger.error("To use this driver, make sure the module python-can is installed")
+            raise ValueError("python-can must be installed.")
+
+        # get bus from default config
+        try:
+            self._bus = can.Bus()
+            # self._bus.state = can.bus.BusState.ACTIVE

```

Comment:
Nitpick: please remove commented code.

Reply:
@windelbouwman this is a valid suggestion, please address.